### DISCUSSION TITLE
Is it possible to enable highlighting in results?

### DISCUSSION THREAD

**Discussion by olivroy** [quarto-dev/quarto-cli#6106] [Q&A] [answered]
### Description

If you look at the results on this pkgdown website, you see that `pillar::style_subtle()`is respected. and NA is red.

https://dplyr.tidyverse.org/reference/filter.html#ref-examples

However, it is not the case with quarto
here is an example: https://r4ds.hadley.nz/numbers.html#minimum-and-maximum

Is it planned to add support for this in the future.
![image](https://github.com/quarto-dev/quarto-cli/assets/52606734/114012f5-caac-4365-9ef2-9d6b6b439c32)

**Comment by cderv** [comment, marked as answer]
**pkgdown** is not using R Markdown nor Pandoc and does a specific processing to restore the ANSI escape color that you get in terminal. 

This is not done directly by **downlit** package, but as another specific step in **pkgdown**

I believe there is already a feature request to bring support in Quarto - either directly in Quarto or through R packages used. (as this is not working either in pkgdown).

* https://github.com/quarto-dev/quarto-cli/issues/3730

Other related issue with more details on how and why: 

* https://github.com/quarto-dev/quarto-cli/discussions/3717
* https://github.com/quarto-dev/quarto-cli/discussions/3957

**Reply by olivroy** [reply]
Thanks much!
